e2-studio-star-rx72n-firmware/tests/mocks/mock_rx_hcsr04_irq.c

```
@@ -0,0 +1,150 @@
+ * ICU Mock Functions
+ * =============================================================================
+ */
+
+/**
+ * @brief Mock: Configure ICU for edge detection (no-op stub)
+ *
+ * @param[in] irq_num  IRQ number (unused in mock)
+ * @param[in] priority Interrupt priority (unused in mock)
+ * @return k_rx_ok always
+ */
+rx_err_t rx_hcsr04_icu_configure(const uint8_t irq_num, const uint8_t priority)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'rx_hcsr04_icu_configure' of similar type ('const uint8_t') are easily swapped by mistake

(bugprone-easily-swappable-parameters)